api/v1alpha1/conversion_test.go

```diff
@@ -68,11 +69,11 @@ func TestFuzzyConversion(t *testing.T) {
 		Spoke: &Storage{},
 	}))
 
-	t.Run("for SystemConfiguration", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
-		Hub:         &dwsv1alpha2.SystemConfiguration{},
-		Spoke:       &SystemConfiguration{},
-		FuzzerFuncs: []fuzzer.FuzzerFuncs{SystemConfigurationFuzzFunc},
-	}))
+	//t.Run("for SystemConfiguration", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
+	//	Hub:         &dwsv1alpha2.SystemConfiguration{},
+	//	Spoke:       &SystemConfiguration{},
+	//	FuzzerFuncs: []fuzzer.FuzzerFuncs{SystemConfigurationFuzzFunc},
+	//}))
```

Comment:
Is leaving this here intentional or do you want to delete it?

Reply:
(unsure...unsure) I want to leave it. If you use a not-elcap-gigantic SystemConfiguration then it still works, and it's our only in-tree example of using FuzzerFuncs for a complex conversion like this one turned out to be.